.tipi/deps

```diff
@@ -0,0 +1,5 @@
+{ 
+  "boostorg/predef" : { "@" : "master" },
+  "ikalnitsky/termcolor" : { "@" : "master" },
```

Comment:
Shouldn't we perhaps pin the versions of these? Does highly accellerate the dev time if we don't pull the latest rev on every single build.

Reply:
For me it doesn't change anything because tipi will always look if the tagger is up to date.
What you are talking about to save time is the -n option.

@daminetreg Do you confirm or not what I say?

Reply:
after discussion it is @pysco68  who is right